src/__tests__/wait-for-element-to-be-removed.js

```diff
@@ -73,9 +74,11 @@ test('it waits for the callback to throw error or a falsy value and only reacts
   for (const [mutationImpl, callbackImpl] of mutationsAndCallbacks) {
     callback.mockImplementation(callbackImpl)
     mutationImpl()
-    await skipSomeTimeForMutationObserver() // eslint-disable-line no-await-in-loop
+    skipSomeTimeForMutationObserver() // eslint-disable-line no-await-in-loop
```

Comment:
Doesn't this still need the await?

Reply:
I don't think so. Now the test runs synchronously because we're using jest fake timers.